### PR TITLE
finalized design, testers for welcome page, icon, headings

### PR DESCRIPTION
**Pull request by jut008**

### CODE REVIEW

**Comment by jut008** [comment]
i tried to make testers for the home page, but i cant seem to make it work because the buttons are images (running into a lot of errors, so i think we can skip home page testers tbh)

**Comment by jut008** [comment]
done! someone please review !

File: client/src/components/pages/welcome.test.tsx
Comment (Shuzhengz) [inline client/src/components/pages/welcome.test.tsx:1]: maybe we should put all tests into a dedicated dir smth like this:

image

File: client/src/components/pages/welcome.tsx
Comment (Shuzhengz) [inline client/src/components/pages/welcome.tsx:1]: idt `React` is used

**Review by Shuzhengz — CHANGES_REQUESTED** [review]
Yoo looks very nice! However I think we should probably change the buttons on the home screen to some other color when hovered on, right now it's the same as background and that's a bit confusing for the users:

image

File: client/src/components/pages/welcome.test.tsx
Comment (jut008) [reply client/src/components/pages/welcome.test.tsx:1]: got it!

File: client/src/components/pages/alarm.tsx
Comment (Shuzhengz) [inline client/src/components/pages/alarm.tsx:1]: not the best practice to modify files outside of scope, will cause merge conflicts to #43

Comment (Shuzhengz) [reply client/src/components/pages/alarm.tsx:1]: prob wanna wait till #43 is merged before merging this to avoid conflicts :(

File: client/src/components/pages/welcome.tsx
Comment (Shuzhengz) [reply client/src/components/pages/welcome.tsx:1]: actually it might be better to keep it

**Review by Shuzhengz — APPROVED** [review]
Nice!

**Review by Shuzhengz — APPROVED** [review]
nice

**Pull request merged**
